docs/ha_3scale.adoc

```diff
@@ -217,4 +217,67 @@ In this setup, to ensure good database access latency, each cluster will have it
 . Manual failovers of the databases to execute are (order does not matter):
 .. AWS RDS: *System* and *Zync*
 .. AWS ElastiCaches: *Backend* and *System*
-. Execute step 4 and 5 from <<manual-failover-shared-databases>>
\ No newline at end of file
+. Execute step 4 and 5 from <<manual-failover-shared-databases>>
+
+=== Active-Active clusters (not supported)
+
+Having an application working on *active/active* configuration is always difficult for any application:
+
+- *Soft Databases limitation*: it is complex but can be achieved with some trade offs, but it is mainly responsibility from the administrator (not 3scale responsibility)
+- *Hard Application limitation*: this one is a hard limitation imposed by how 3scale manages OpenShift Routes and cannot be achieved
+
+==== Soft Databases limitation (out of 3scale scope)
+
+One of the most difficult parts for any application deployed on *active*/*active* (not only 3scale), is having databases on *active*/*active* mode, and in case of 3scale there are many different databases involved which would require different *active*/*active* implementations for each of them:
+
+- System-mysql (or system-postgres or system-oracle, you can choose)
+- System-redis
+- Backend-redis
+- Zync-database (postgresql)
+- system-sphinx (doesn't need HA)
+- system-memcached (doesn't need HA)
+
+So, from 3scale point of view, an administrator need to ensure that database connection strings configured in 3scale APIManager custom resource on any OCP cluster will be always a master instance (allowing write operations).
```

Comment:
Do we know that the db clients we use in 3scale are compatible with the active/active soluitons these DB offerings provide? For example MySQL's Galera cluster or MariaDB and postgres crucnhy DB.

Obviously for Redis we had to upgrade our redis clients to support redis sentinels and redis enterprise but maybe the same is not true for the other DBs we support and their respective offerings that support Active/Active.

Reply:
I have checked that most of this A/A DB solutions like mysql galera o postgres cruncyDB:
- Can be accessed with smart db clients by adding your database instances on the connection string (possibly not supported on db clients used by 3scale)
- Can deploy official TCP smart load balancers (galera LB, haproxy, nginx...) in front of DB nodes, so there is a single shared LB across your DB nodes that will choose the less loaded backend DB node (less connections...)... So from 3scale point of view any standard db client should work by providing a single DB hostname (the Load Balancer)